frontend/src/app/shared/common.ts

```diff
@@ -34,7 +34,7 @@ export function optionalValue(param: object): { [p: string]: any } {
       }),
   );
 }
-export function isInValid(baseline: number, stretchGoal: number, value: number): boolean {
+export function isLastCheckInNegative(baseline: number, stretchGoal: number, value: number): boolean {
   if (value < baseline && baseline <= stretchGoal) return true;
   return value > baseline && baseline > stretchGoal;
```

Comment:
Please Adjust for case Baseline equals Value in this case return true in order to show fail in Overview.